### PR TITLE
Add snippet of participants to search result

### PR DESCRIPTION
**Pull request by okkez**
Close https://github.com/bm-sms/daimon-news-misc/issues/88

### CODE REVIEW

File: app/decorators/post_search_result_set_decorator.rb [hunk 1]
```diff
@@ -47,7 +47,12 @@ def to_og_params
   end
 
   def excerpt(post)
-    snippets = snippet(post.plain_text_body, class: "search-result__keyword")
+    snippets = []
+    post.credits.each do |credit|
+      snippets.concat(snippet(credit.participant.name, class: "search-result__keyword"))
```
Comment (myokoym) [inline app/decorators/post_search_result_set_decorator.rb:52]: :cool:

File: test/integration/search_test.rb [hunk 2]
```diff
@@ -273,17 +273,29 @@ class SearchTest < ActionDispatch::IntegrationTest
       end
     end
 
-    test "display snippets when snippets is not empty" do
+    data(
+      "title" => {
+        query: "title",
+        snippets: ["title", "title"]
```
Comment (myokoym) [inline test/integration/search_test.rb]: I think they are not snippets. How about `keywords_in_snippet`?

Comment (okkez) [inline test/integration/search_test.rb]: How about `highlighted`?

Comment (myokoym) [inline test/integration/search_test.rb]: Sounds good!

**Comment by myokoym** [comment]
Looks good :stars:

**Comment by myokoym** [comment]
:airplane:

**Pull request merged**